recipe/vs_snprintf.patch

```diff
@@ -0,0 +1,14 @@
+diff --git Source/LibTIFF4/tif_config.h Source/LibTIFF4/tif_config.h
+index 64391b8..252b94d 100644
+--- Source/LibTIFF4/tif_config.h
++++ Source/LibTIFF4/tif_config.h
+@@ -81,7 +81,9 @@ If your big endian system isn't being detected, add an OS specific check
+ #endif // BYTE_ORDER
+ 
+ #ifdef _WIN32
++#if defined(_MSC_VER) && _MSC_VER < 1900
```

Comment:
VS 2015 (== _MSC_VER 1900) finally added C99 standard support.  This code was a workaround that causes a conflict on VS 2015 (python 3.5).

https://msdn.microsoft.com/en-us/library/hh409293.aspx#BK_CRT